tests/runtime/call

```diff
@@ -412,3 +412,21 @@ EXPECT OK
 REQUIRES_FEATURE skboutput
 TIMEOUT 5
 MIN_KERNEL 5.5
+
+NAME debugf
+BEFORE ./testprogs/empty_trace_buffer
```

Comment:
It's not very practical to write a separate program for this when you can just use `echo > /sys/kernel/debug/tracing/trace`. I know I suggested to use the `BEFORE` directive, but since it doesn't allow running programs with arguments, the original version (running the `echo` command as a part of `RUN`) seems much better to me.